src/unix/stream.c

```diff
@@ -1602,6 +1602,32 @@ int uv_read_stop(uv_stream_t* stream) {
   return 0;
 }
 
+int uv_export(uv_stream_t* stream, uv_stream_info_t* info) {
+  int fd;
+
+  if (stream->type != UV_TCP)
+    return UV_EINVAL;
+
+  fd = uv__dup(stream->io_watcher.fd);
```

Comment:
```suggestion
  fd = uv__dup2_cloexec(stream->io_watcher.fd);
```
(and remove `uv__dup()`)